internal/preferences.go

```diff
@@ -41,7 +41,10 @@ func (p *InMemoryPreferences) BoolListWithFallback(key string, fallback []bool)
 	if !ok {
 		return fallback
 	}
-
+	_, ok1 := value.([]bool)
+	if !ok1 {
+		return fallback
+	}
 	return value.([]bool)
```

Comment:
You are doing a second cast here which is unnecessary work. Just save the first parameter of the `_, ok` call above (replacing `_` with `value`).